src/core/evaluator.js

```diff
@@ -1009,16 +1009,23 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
           // var x = pt[0];
           // var y = pt[1];
 
+          var charSpacing = 0;
+          if (i > 0 || textChunk.str.length > 0) {
```

Comment:
When I first created the issue, I had started testing on the `i > 0` and quickly found within a TJ, that was getting called multiple times, so I added the `textChunk.str` check. But since issue only occurs within a TextChunk anyway, that initial check for `i` seemed superfluous. Just thought I would mention where I ended up since I had found it initially.